mellanox-ofed/pkg.yaml

```diff
@@ -1,5 +1,5 @@
 name: mellanox-ofed-pkg
-variant: scratch
+variant: alpine
```

Comment:
Not sure whether or not we should do so. Why doesn't it work without `variant: alpine`, what's missing in our tools, and why not add it there?